slapp/explorer/regions.py

```diff
@@ -163,3 +201,388 @@ def municipalities_details(ids: list[int]) -> list[Municipality]:
+                    {"source": "Netz ", "target": "Grünheide (Mark)", "value": 100},
+                    {"source": "Erkner", "target": "Netz  ", "value": 120},
+                    {"source": "Netz  ", "target": "Erkner", "value": 20},
+                    {"source": "Rüdersdorf bei Berlin", "target": "Netz   ", "value": 100},
+                    {"source": "Netz   ", "target": "Rüdersdorf bei Berlin", "value": 40},
+                ],
+                "nodes": [
+                    {"name": "Strausberg", "x": 50, "y": -50, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {
+                        "name": "Rüdersdorf bei Berlin",
+                        "x": 0,
+                        "y": 0,
+                        "itemStyle": {"color": "#798897"},
+                        "symbolSize": 30,
+                    },
+                    {
+                        "name": "Grünheide (Mark)",
+                        "x": 50,
+                        "y": 50,
+                        "itemStyle": {"color": "#798897"},
+                        "symbolSize": 30,
+                    },
+                    {"name": "Erkner", "x": -10, "y": 50, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 80, "y": -25, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz ", "x": 60, "y": 75, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz  ", "x": -20, "y": 75, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                    {"name": "Netz   ", "x": -20, "y": -20, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+        ]
+    elif region == "Kiel":
+        energy_data = [
+            {
+                "title": "Stromaustausch (Kiel)",
+                "energyData": [
+                    {"source": "Kiel", "target": "Netz", "value": 120},
+                    {"source": "Netz", "target": "Kiel", "value": 20},
+                ],
+                "nodes": [
+                    {"name": "Kiel", "x": 0, "y": 0, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 25, "y": 0, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+            {
+                "title": "Stromaustausch (Kiel)",
+                "energyData": [
+                    {"source": "Kiel", "target": "Netz", "value": 120},
+                    {"source": "Netz", "target": "Kiel", "value": 20},
+                ],
+                "nodes": [
+                    {"name": "Kiel", "x": 0, "y": 0, "itemStyle": {"color": "#798897"}, "symbolSize": 30},
+                    {"name": "Netz", "x": 25, "y": 0, "itemStyle": {"color": "#000000"}, "symbolSize": 10},
+                ],
+            },
+        ]
+    return energy_data
+
+
+def flow_chart(request: HttpRequest) -> JsonResponse:
```

Comment:
we should move views to views.py.
If view logic is short, we can keep it in views.py, otherwise we could move it to regions.py and import it in related view.